compiler/il/OMRILOps.cpp

```diff
@@ -58,6 +58,8 @@ OMR::ILOpCode::checkILOpArrayLengths()
 
       TR_ASSERT(props.opcode == opCode, "_opCodeProperties table out of sync at index %d, has %s\n", i, op.getName());
       }
+
+   TR_ASSERT_FATAL(_opCodeProperties[TR::viRegLoad].dataType == OMR::DataType::createVectorType(TR::Int32, TR::VectorLength128), "Vector type was not initialized correctly\n");
```

Comment:
Should this be `TR::DataType::createVectorType`? (rather than `OMR::...`)

Reply:
Actually, I think it's OMR::

Reply:
Reflecting here our discussion about the rationale for this change: `DataType` is extensible. Definitions in the base OMR layer are within `OMR::DataType`, but consumers should (and do) generally use `TR::DataType`, which is the fully derived type